examples/run_k_grid.py

```diff
@@ -6,15 +6,18 @@
 
 def test_run_k_grid():
     from carmm.run.aims_calculator import get_k_grid
+    from ase.build import molecule
+
 
     #### Traditional ASE functionality #####
     from data.model_gen import get_example_slab as slab
     slab = slab(adsorbate=True)
     #########
     sampling_density = 0.02 # example value of sampling density /Angstrom
     k_grid = get_k_grid(slab, sampling_density, surface=True, verbose=True )
-    assert(k_grid[0] == 6)
-    assert(k_grid[1] == 6)
-    assert(k_grid[2] == 1)
+    assert k_grid[0] == 6
+    assert k_grid[1] == 6
+    assert k_grid[2] == 1
```

Comment:
All good. Could be shortened to this (but only minor)

```suggestion
    assert k_grid == [6, 6, 1]
```

Reply:
Good to merge whether you make this change or not.

Reply:
`assert k_grid == [6, 6, 1]` should read  `assert k_grid == [6, 6, 1]` as returned `k_grid` is a tuple